tests/ternaries/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1633,8 +2301,8 @@ a => a ? aasdasdasdasdasdasdaaasdasdasdasdasdasdasdasdasdasdasdasdasdaaaaaa : a
 debug ? (this.state.isVisible ? "partially visible" : "hidden") : null;
 debug
 	? this.state.isVisible && somethingComplex
-		? "partially visible"
-		: "hidden"
+	  ? "partially visible"
+	  : "hidden"
```

Comment:
This shouldn't be changed, we have a special case for ternary in ternary to use tabs in `--use-tabs` mode, that's what the removed part in 3018dd1ff5c2669dff0b3feb593f4ca9ba526175 wanted to achieve.

Reply:
Ok, thanks. I will fix it